packages/mongodb-memory-server-core/src/util/__tests__/dryBinary.test.ts

```diff
@@ -155,6 +157,46 @@ describe('DryBinary', () => {
     });
   });
 
+  describe('getBinaryName', () => {
+    afterEach(() => {
+      delete process.env[envName(ResolveConfigVariables.USE_ARCHIVE_NAME_FOR_BINARY_NAME)];
+    });
+
+    it('should return archive style ', async () => {
+      process.env[envName(ResolveConfigVariables.USE_ARCHIVE_NAME_FOR_BINARY_NAME)] = 'true';
+
+      expect(
+        await DryMongoBinary.getBinaryName({
```

Comment:
because `getos` is now a call in this function, it should probably have at least one spy attached to see that it is not actually called when providing `os`